sentry_sdk/integrations/aws_lambda.py

```diff
@@ -28,8 +29,8 @@
     F = TypeVar("F", bound=Callable[..., Any])
 
 # Constants
-TIMEOUT_THRESHOLD_MILLIS = 1500  # Minimum time required to capture TimeoutError
-SECONDS_CONVERSION_FACTOR = 1000.0
+TIMEOUT_WARNING_BUFFER = 1500  # Buffer time required to send timeout warning to Sentry
+MILLIS_TO_SECONDS = 1000.0
 
 
 def _wrap_init_error(init_error):
```

Comment:
@rhcarvalho @untitaker in this method see line no. 57, the event is not captured from an actual exception.

Reply:
I'm suggesting we make this change below (this change works):-

            `.if error.get("errorType"):
            .    try:
            .        exc_info = sys.exc_info()
            .        reraise(*exc_info)
            .    except Exception:
            .        exc_info = sys.exc_info()
            .        event, hint = event_from_exception(
            .            exc_info,
            .            client_options=client.options,
            .            mechanism={"type": "aws_lambda", "handled": False},
            .        )
            .    hub.capture_event(event, hint=hint)`